tck/src/main/java/org/eclipse/microprofile/rest/client/tck/interfaces/ClientHeaderParamClient.java

```diff
@@ -73,7 +72,7 @@
     @GET
     @ClientHeaderParam(name="OptionalMethodHeader", value="{fail}", required=false)
     @ClientHeaderParam(name="MethodHeaderExplicit", value="SomeValue")
-    Map<String, String> methodOptionalMethodHeaderNotSentWhenComputeThrowsException();
+    JsonObject methodOptionalMethodHeaderNotSentWhenComputeThrowsException();
```

Comment:
RestEasy doesn't want to work with Map here, if there's a reason why Map should be "usable" here, I will revert this part.